app/controllers/application_controller.rb

```diff
@@ -1,2 +1,19 @@
 class ApplicationController < ActionController::API
+
```

Comment:
I already understand the reasoning behind this, but it would be helpful to document why you chose to put this in the Application Controller instead of the Company Controller. Could you explain the thought process behind this change?

Reply:
Application Controller allows for any of the other controllers that need some form of authentication to utilize the function. All we'll have to add is the `before action:authenticate_user`